### PR TITLE
[CC]Amount / 100 for tps since amount is send in pennies and added in…

### PR DESCRIPTION
**Pull request by cavster**
… account to end of text for some of the rows

### CODE REVIEW

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/onlinepaymentservice/AmountFormatter.scala [hunk 1]
```diff
@@ -32,6 +32,7 @@ object AmountFormatter {
   }
 
   def formatAmount(amount: String): String = formatter.format(toDouble(amount))
+  def formatAmountPennies(amount: String): String = formatter.format(toDouble(amount)/100)
```
Comment (andy1138) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/onlinepaymentservice/AmountFormatter.scala]: Should be formatAmountPence or formatAmountInPence

Comment (cavster) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/onlinepaymentservice/AmountFormatter.scala]: It's for formmating an amount that is in pennies to be shown in pounds

Comment (cavster) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/onlinepaymentservice/AmountFormatter.scala]: I changed it please merge

Comment (andy1138) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/onlinepaymentservice/AmountFormatter.scala]: pennies still sounds wrong, please suggest something else

Comment (cavster) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/onlinepaymentservice/AmountFormatter.scala]: I changed it check the latest version

Comment (andy1138) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/onlinepaymentservice/AmountFormatter.scala]: 👍 good for me, you will need to change the other places before I merge

Comment (cavster) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/onlinepaymentservice/AmountFormatter.scala]: Lol done now

Comment (andy1138) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/onlinepaymentservice/AmountFormatter.scala]: :-)

**Pull request merged**